qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -77,8 +77,12 @@ def wire_symbol(
         self, reg: Optional['Register'], idx: Tuple[int, ...] = tuple()
     ) -> 'WireSymbol':
         if reg is None:
-            return Text(f'x<{self.less_than_val}')
-        return super().wire_symbol(reg, idx)
+            return Text("")
+        if reg.name == 'x':
+            return TextBox("x")
+        if reg.name == 'target':
+            return TextBox("z ^ (x < a)")
```

Comment:
should probably omit the spacing around operators as horizontal space is usually at a premium in these boxes/diagrams